### PR TITLE
Period configuration

### PR DESCRIPTION
**Pull request by ares**
Depends on https://github.com/OpenSCAP/scaptimony/pull/15

### CODE REVIEW

**Comment by isimluk** [comment]
I am getting:
```
ActiveRecord::RecordInvalid: Validation failed: Default value is invalid
```
on `policies_param.save!` during `ensure_needed_puppetclasses`

**Comment by ares** [comment]
Strange, it seems that it does not like the ERB, try applying https://github.com/theforeman/foreman/pull/2080 to your Foreman (which is needed for right ENC format anyway). What foreman version do you test it on?

File: app/views/scaptimony_policies/steps/_schedule_form.html.erb [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
-  <%= select_f(f, :period, %w[Weekly Monthly], :downcase, :to_s, :include_blank => _('Choose period')) %>
+  <%= select_f(f, :period, %w[Weekly Monthly Custom], :downcase, :to_s,
+               { :include_blank => _('Choose period') },
+               { :onchange => 'period_selected(this)' }) %>
   <%= select_f(f, :weekday, Date::DAYNAMES, :downcase, :to_s, :include_blank => _('Choose weekday')) %>
+  <%= text_f(f, :day_of_month, :help_inline => _('Number of a day in month, note that not all months have same count of days')) %>
+  <%= text_f(f, :cron_line, :help_inline => _('You can specify custom cron line, e.g. "0  3  *  *  *", separate each of 5 values by space')) %>
 </div>
+
+<script>
```
Comment (shlomizadok) [inline app/views/scaptimony_policies/steps/_schedule_form.html.erb]: Should be on a js file?

Comment (ares) [inline app/views/scaptimony_policies/steps/_schedule_form.html.erb]: fixed, thanks

**Comment by ares** [comment]
Anything blocking this PR? Did the Foreman patch solve the issue?

**Comment by isimluk** [comment]
I have got this working finally. 

Merging into 0.3.0. Thank You very much @ares!

**Pull request closed without merging**